security/org-account-context/main.tf

```diff
@@ -276,3 +276,368 @@ module "github_oidc_provider" {
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup"
+                value = "true"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup_retention"
+                value = "30days"
+              }
+            ]
+          }
+        },
+        {
+          name = "select-uat-30-days"
+          resources = [
+            "arn:aws:dynamodb:*:*:table/*",
+            "arn:aws:rds:*:*:db:*"
+          ]
+          conditions = {
+            string_equals = [
+              {
+                key   = "aws:ResourceTag/dfds.env"
+                value = "uat"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup"
+                value = "true"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup_retention"
+                value = "30days"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      plan_name = "60-days-retention"
+      rules = [
+        {
+          name = "weekly-60-days-retention"
+          schedule = "cron(0 3 ? * L *)"
+          enable_continuous_backup = false
+          lifecycle = {
+            delete_after = 60
+          }
+        }
+      ]
+      selections = [
+        {
+          name = "select-prod-60-days"
+          resources = [
+            "arn:aws:dynamodb:*:*:table/*",
+            "arn:aws:rds:*:*:db:*"
+          ]
+          conditions = {
+            string_equals = [
+              {
+                key   = "aws:ResourceTag/dfds.env"
+                value = "prod"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup"
+                value = "true"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup_retention"
+                value = "60days"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      plan_name = "180-days-retention"
+      rules = [
+        {
+          name                     = "monthly-180-days-retention"
+          schedule                 = "cron(0 3 L * ? *)"
+          enable_continuous_backup = false
+          lifecycle = {
+            delete_after = 180
+            cold_storage_after = 90 # Cold storage only stores full backups
+          }
+        }
+      ]
+      selections = [
+        {
+          name      = "select-prod-180-days"
+          resources = [
+            "arn:aws:rds:*:*:db:*",
+            "arn:aws:dynamodb:*:*:table/*"
+          ]
+          conditions = {
+            string_equals = [
+              {
+                key   = "aws:ResourceTag/dfds.env"
+                value = "prod"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup"
+                value = "true"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup_retention"
```

Comment:
The ResourceTag should be a variable since infrastructure-module is an open source project, and I guess any external user would prefer to specify their own ResourceTag name,